utils/types/elective.ts

```diff
@@ -0,0 +1,33 @@
+import { MultiLangString } from "@/utils/types/common";
+import { Teacher } from "@/utils/types/person";
+import { SubjectGroup } from "@/utils/types/subject";
+import { Student } from "@/utils/types/person";
+import { Classroom } from "@/utils/types/classroom";
+
+export type ElectiveSubject = {
```

Comment:
Don't forget the `session_code` property naa

Reply:
Is this line wrong?

https://github.com/suankularb-wittayalai-school/mysk-frontend/blob/82b0d1365313f7bf163953d90aafca4a0231454e/utils/types/elective.ts#L9

Reply:
I did not see that somehow